Tests/FluentAssertions.Equivalency.Specs/DictionarySpecs.cs

```diff
@@ -1170,4 +1170,22 @@ public void When_a_custom_rule_is_applied_on_a_dictionary_it_should_apply_it_on_
             .WhenTypeIs<double>()
         );
     }
+
+    [Fact]
+    public void Passing_the_reason_to_the_inner_equivalency_assertion_works()
+    {
+        var subject = new Dictionary<string, object>
+        {
+            ["a"] = new List<int>()
+        };
+
+        var expected = new Dictionary<string, object>
+        {
+            ["a"] = new List<int> { 42 }
+        };
+
+        Action act = () => subject.Should().BeEquivalentTo(expected, "FOO");
+
+        act.Should().Throw<XunitException>().WithMessage("*FOO*");
```

Comment:
To ensure we pass both `because` and `becauseArgs` all the way.
```suggestion
        Action act = () => subject.Should().BeEquivalentTo(expected, "FOO {1}", "BAR");

        act.Should().Throw<XunitException>().WithMessage("*FOO BAR*");
```

Reply:
I think this should be `{0}`, isn't it? 😉

Reply:
🤐